### PR TITLE
Add Hexiar (BEP20)

### PR DESCRIPTION
**Pull request by borgilkh**
Adding info for BEP20 token 'Hexiar'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by mmm0171** [comment]
0xa36fD8fF0524f287422337d99bABdf05923f581B

File: blockchains/ardcoin:info/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "name": "ArdCoin",
+    "symbol": "ARDX",
+    "type": "ERC20",
+    "decimals": 2,
+    "description": "ArdCoin will serve as a blockchain-based loyalty program for all existing and future services and products of Ard Financial Group (Ard Holdings), its subsidiaries and ArdCoin merchants.",
+    "website": "https://ardcoin.com",
+    "explorer": "https://etherscan.io/token/0xB8E2e2101eD11e9138803cd3e06e16dd19910647",
+    "status": "active",
+    "id": "0xB8E2e2101eD11e9138803cd3e06e16dd19910647"
+}
```
Comment (mmm0171) [inline blockchains/ardcoin:info/info.json]: 0xa36fD8fF0524f287422337d99bABdf05923f581B

**Review by mmm0171 — APPROVED** [review]
0xa36fD8fF0524f287422337d99bABdf05923f581B

**Comment by Cryptocool1** [comment]
This is a BEp20 token, correct folder is blockchains/smartchain/assets.

Please create new PR, follow doc https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863
You may want to try our Assets app in beta; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you.
See https://community.trustwallet.com/t/how-to-submit-a-token-logo-using-assets-app/82957.

**Pull request closed without merging**